### PR TITLE
Add Bitcoin kit to demos page

### LINKED ISSUE #675: Add Bitcoin Kit demos page link
### Feature or change description

See design [here](https://www.figma.com/design/4fVd9wneclsvYDYD95ApZ9/Hemi-Portal?node-id=4375-4070&node-type=frame&m=dev)

URL is https://bitcoin-kit.hemi.xyz/

### PR DESCRIPTION
**Pull request by gndelia**
### Description

This PR adds the Bitcoin Kit Demo to the `/demos` page. I also sorted a few keys in the translation files

### Screenshots

image

image

image

### Related issue(s)

Closes #675

### Checklist

- [x] Manual testing passed.
- [ ] Automated tests added, or N/A.
- [ ] Documentation updated, or N/A.
- [ ] Environment variables set in CI, or N/A.

### CODE REVIEW

File: webapp/messages/en.json [hunk 1]
```diff
@@ -60,20 +60,24 @@
     }
   },
   "demos": {
-    "heading": "Demos",
-    "sub-heading": "Learn and explore with Hemi developer demos",
-    "hemihatchlings": {
-      "heading": "Hemi Hatchlings",
+    "bitcoin-kit": {
+      "heading": "Hemi Bitcoin Kit Demo",
       "sub-heading": "Learn how Ethereum & Bitcoin finality works"
```
Comment (ArturDolzan) [inline webapp/messages/en.json:65]: Should we use the same sub-heading as we have used for hemihatchlings?

Comment (gndelia) [reply webapp/messages/en.json:65]: It's the one from the design, but you're correct, it doesn't sound like it's the one. Maybe @rmilrad  could provide the appropriate texts once he's back

Comment (gndelia) [reply webapp/messages/en.json]: Wording updated here [9193980](https://github.com/hemilabs/ui-monorepo/pull/722/commits/9193980ea9c77797be65747adfbe5547bdd6dee7)

**Review by ArturDolzan — APPROVED** [review]
LGTM

**Comment by dvnahuel** [comment]
LGTM

**Review by gabmontes — APPROVED** [review]
I thought the fade effect in bitcoin_kit_large.png could have been done with CSS but 🤷🏼‍♂️

**Comment by gndelia** [comment]
> I thought the fade effect in bitcoin_kit_large.png could have been done with CSS but 🤷🏼‍♂️

I had a similar thought at the beginning, but that implied adding custom code for custom cards... I felt it was more straightforward to just have the static image with the blur effect.

**Pull request merged**